File: ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/SdsExtrapolationMode.cs
// <copyright file="SdsExtrapolationMode.cs" company="OSIsoft, LLC">
//
// </copyright>

using System;
using System.Collections.Generic;
using System.Text;

namespace SdsRestApiCore
{
    public enum SdsExtrapolationMode : int
    {
        All = 0,
        None = 1,
        Forward = 2,
        Backward = 3
    }
}

File: ocs_samples/library_samples/Python3/ocs_sample_library_preview/SDS/SdsExtrapolationMode.py
# SdsStreamExtrapolationMode.py
#

from enum import Enum

class SdsExtrapolationMode(Enum):
    """
    SdsStreamExtrapolationMode 0 -3
    """
    All = 0
    Nonex = 1
    Forward = 2
    Backward = 3


